styles/services.scss

```diff
@@ -2,3 +2,4 @@
 @import "~@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-segmented/index";
 
```

Comment:
think you guys might need to import the `content-block-cards.scss` file here

Reply:
I didn't import this file, because it isn't on the story book [Content Block Cards - Story Book](http://ibmdotcom-react.mybluemix.net/?path=/docs/patterns-blocks-contentblockcards--default)

Reply:
oh gotcha, I'll open a PR to update the documentation

Reply:
Thanks @annawen1.